packages/react/src/header/Header.stories.tsx

```diff
@@ -1,154 +1,101 @@
-	logo: undefined,
-	size: 'sm',
+export const Small: Story = {
+	args: {
+		heading: 'Internal application',
+		subline: undefined,
+		size: 'sm',
+	},
 };
 
-export const LongSubline = Template.bind({});
-LongSubline.args = {
-	...defaultArgs,
-	subline: 'Service description that could be a little longer',
+export const SmallNoLogo: Story = {
+	args: {
+		heading: 'Internal application',
+		subline: undefined,
+		logo: undefined,
+		size: 'sm',
+	},
 };
 
-export const Badge = Template.bind({});
-Badge.args = {
-	...defaultArgs,
-	badgeLabel: 'Beta',
+export const LongSubline: Story = {
+	args: {
+		subline: 'Service description that could be a little longer',
+	},
 };
 
-export const NoLogo = Template.bind({});
-NoLogo.args = {
-	...defaultArgs,
-	logo: undefined,
+export const NoLogo: Story = {
+	args: {
+		logo: undefined,
+	},
 };
 
-export const Search = Template.bind({});
-Search.args = {
-	...defaultArgs,
-	rightContent: (
-		<SearchBox onSubmit={console.log}>
-			<SearchBoxInput label="Search this website" />
-			<SearchBoxButton iconOnly={{ xs: true, md: false }}>
-				Search
-			</SearchBoxButton>
-		</SearchBox>
-	),
+export const WithSearchBox: Story = {
+	args: {
+		rightContent: (
+			<SearchBox onSubmit={console.log}>
+				<SearchBoxInput label="Search this website" />
+				<SearchBoxButton iconOnly={{ xs: true, md: false }}>
+					Search
+				</SearchBoxButton>
+			</SearchBox>
+		),
+	},
 };
-Search.storyName = 'With SearchBox';
 
-const TemplateWithMainNav: ComponentStory<typeof Header> = (args) => {
-	return (
-		<Box>
+export const WithMainNav: Story = {
+	args: {},
+	render: (args) => (
+		<Fragment>
 			<Header {...args} />
 			<MainNav
+				activePath="#account"
 				items={[
-					{
-						href: '#account',
-						label: 'Home',
-					},
-					{
-						href: '#establishments',
-						label: 'Establishments',
-					},
-					{
-						href: '#intelligence',
-						label: 'Data and insights',
-					},
-					{
-						href: '#compliance',
-						label: 'Compliance',
-					},
+					{ href: '#account', label: 'Home' },
+					{ href: '#establishments', label: 'Establishments' },
+					{ href: '#intelligence', label: 'Data and insights' },
+					{ href: '#compliance', label: 'Compliance' },
 				]}
-				activePath="#account"
 				background="body"
 			/>
-		</Box>
-	);
-};
-
-export const WithMainNav = TemplateWithMainNav.bind({});
-WithMainNav.storyName = 'With MainNav';
-WithMainNav.args = {
-	...defaultArgs,
-	background: 'bodyAlt',
-};
-
-export const WithMainNavAndSearch = TemplateWithMainNav.bind({});
-WithMainNavAndSearch.storyName = 'With MainNav and SearchBox';
-WithMainNavAndSearch.args = {
-	...defaultArgs,
-	rightContent: (
-		<SearchBox onSubmit={console.log}>
-			<SearchBoxInput label="Search this website" />
-			<SearchBoxButton iconOnly={{ xs: true, md: false }}>
-				Search
-			</SearchBoxButton>
-		</SearchBox>
+		</Fragment>
 	),
-	background: 'bodyAlt',
 };
 
-export const CoBranded = Template.bind({});
```

Comment:
This story has been removed. It's been here for 1.5years+

Reply:
what if we need to cobrand for STS work?